lib/src/client/http2_connection.dart

```diff
@@ -319,7 +315,7 @@ class _SocketTransportConnector implements ClientTransportConnector {
   final Object _host;
   final int _port;
   final ChannelOptions _options;
-  Socket _socket;
+  Socket? _socket; // ignore: close_sinks
```

Comment:
this should probably be `late Socket _socket`.

Reply:
Done